FreeRTOS/iot-reference-nxp-rt1060
# Update 202407.00 release information

Description
-----------
* Update changelog for LTS 202406.00 information
* Update README
* Remove idt_config folder
* Add release.yml file

Test Steps
-----------
Test release in forked repo : https://github.com/chinglee-iot/iot-reference-nxp-rt1060/releases/tag/202407.00-LTS

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
